metals/src/main/scala/scala/meta/internal/metals/testProvider/BuildTargetUpdate.scala

```diff
@@ -68,5 +68,14 @@ object TestExplorerEvent {
 // Represents a single test within a test suite
 final case class TestCaseEntry(
     name: String,
+    displayName: String,
+    runName: String,
```

Comment:
Do we need 3 different names? Ideally name would be the same as runName so that if anyone implemented the protocol extension this would also fix it for them.

And we wouldn't need an update to VS Code to make it work, just to make it display nicer

Reply:
That we can do. 👍